CHANGES.rst

```diff
@@ -1,20 +1,32 @@
 Changes
 =======
 
-Version 0.7.4 (UNRELEASED)
+Version 0.7.4 (2021-07-05)
```

Comment:
Everything released except Helm chart.  Need to do also:
- prepare release blog post
- prepare www.reana.io home page update
- test helm chart with researcher-oriented installation method